docs/source/conf.py

```diff
@@ -22,8 +22,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'qcelemental'
-copyright = '2018, The Molecular Sciences Software Institute'
-author = 'The Molecular Sciences Software Institute'
+copyright = '2018-2019, The Molecular Sciences Software Institute'
```

Comment:
`f"Copyright 2018-{datetime.datetime.today().year}"` after `import datetime` or the QCA stack manager is going to start to dread the new year